qr-generator/src/main/java/com/example/qrgenerator/QrGeneratorApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.example.qrgenerator;
```

Comment:
`example` is incorrect. Check how other projects were created.
Most likely, `com.qpay.qrgenerator`